src/common/chat/command_parser.ts

```diff
@@ -8,6 +8,7 @@ export enum CommandType {
   Attack = '!attack',
   Load = '!load',
   Help = '!help',
+  Lookup = '!lookup',
```

Comment:
Pretty minor, but thoughts on making the command !spell ? 

That way for other lookups (weapons or items I think we'll add in the future) the name wouldn't be too general.

Reply:
My initial thought was that those would share !lookup. So regardless of what piece of DnD info you were looking for, it's the same command. I don't have a strong preference though, and !spell is more overt out of the gate - so I'm good to update